bindata/oauth-openshift/deployment.yaml

```diff
@@ -62,6 +62,8 @@ spec:
             - name: https
               containerPort: 6443
               protocol: TCP
+          securityContext:
+            readOnlyRootFilesystem: false # because of the `rm` in args
```

Comment:
you mean the cp

Reply:
right, the cp does rm first by `-f` which I believe is the error we're seeing, let me fix that

Reply:
...or I just confused myself thinking of the removal, anyway

Reply:
fixed